docs/examples/writing_drivers/Creating-Instrument-Drivers.ipynb

```diff
@@ -95,6 +95,62 @@
     "`InstrumentChannel` is a subclass of `InstrumentModule` which behaves identically to `InstrumentModule` you should chose either one depending on if you are implementing a module or a channel. As a rule of thumb you should use `InstrumentChannel` for something that the instrument has more than one of.\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Naming Instruments \n",
+    "\n",
+    "We are aiming to organize drivers in QCoDeS in a consistent way for easy discovery.\n",
+    "Note that not all drivers in QCoDeS are currently named consistently. \n",
+    "However, we aim to gradually update all drivers to be named as outlined above and any new driver \n",
+    "should be named in the way outlined below.\n",
+    "\n",
+    "The same rules should apply for QCoDeS-contrib-drivers with the exception that all drivers are stored in subfolders of the drivers folder. \n",
+    "\n",
+    "### Naming the Instrument class\n",
+    "A driver for an instrument with model `Model` and from the vendor `Vendor` should be stored in the file:\n",
+    "\n",
+    "```\n",
+    "qcodes\\instrument_drivers\\{Vendor}\\{Vendor{_{Model}.py \n",
+    "```\n",
+    "using snake case with an underscore between the vendor and model name but starting the\n",
+    "vendor name with upper case.\n",
+    "\n",
+    "The primary instrument class should be named as follows:\n",
+    "```\n",
+    "class {Vendor}{Model}\n",
+    "     ...\n",
+    "```\n",
+    "E.g Vendor followed by Model number in CamelCase.\n",
+    "\n",
+    "Note that we use vendor names starting with upper case for both folders and file names.\n",
+    "\n",
+    "As an example the driver for the Weinschel 8320 should be stored in the file `qcodes\\instrument_drivers\\Weinschel\\Weinschel_8320.py` and the \n",
+    "class named `Weinschel8320` \n",
+    "\n",
```

Comment:
shall we add known short-enings for Vendors for use in instrument class names? e.g ZI is fine for Zurich Instrument, and NI is fine for National Instruments, AMI is fine for American Magnetics, etc ?